docs/troubleshooting/Performance.mdx

```diff
@@ -79,6 +79,107 @@ module.exports = {
 
 See [Glob pattern in parser's option "project" slows down linting](https://github.com/typescript-eslint/typescript-eslint/issues/2611) for more details.
 
+## Changes to `extraFileExtensions` with `projectService`
+
+Changes to [`extraFileExtensions`](../packages/Parser.mdx#extrafileextensions) between files in the same project with
```

Comment:
[Docs] Nice tip I've picked up, using different wording in the header vs. the body. The two tend to have different kind of phrasing, and this body text is a bit more precise.

```suggestion
Using a different `extraFileExtensions`](../packages/Parser.mdx#extrafileextensions) between files in the same project with
```